### PR TITLE
fix: Add missing variable and correct f-strings

### PR DESCRIPTION
**Pull request by Brian-McHugh**
See commented lines for changes to ex6.py:
- add types_of_people variable assigment
- change variable from 10 to types_of_people
- add letter f before f-strings
- omit unnecessary periods

### CODE REVIEW

File: ex6.py [hunk 1]
```diff
@@ -1,13 +1,19 @@
-x = f"There are {10} types of people."
+# left out assignment for types_of_people mentioned in intro
+types_of_people = 10
```
Comment (josharcheruk) [inline ex6.py:2]: This doesn't match up with the original Python 2 version and I believe the intent here is to indicate that you can add 'digit' strings (or whatever the Python 3.6 equivalent is). 

It's not as clear as the original so may need some tweaking but I'm not sure adding it as a variable 'types_of_people' matches the original intent. 

EDIT: I should clarify that this is necessarily a comment for @Brian-Leary. @zedshaw might simply need to clarify the intent of this exact part as to whether it's going to diverge slightly from the original.

**Comment by zedshaw** [comment]
Great! Yeah, I think using a variable is better there, although PuffinBlue is right that I should show how to just put python expressions in there.

**Comment by Brian-McHugh** [comment]
@zedshaw, happy to help. Please include me in the running for the bug bounty. Would love to win a painting from you. Thanks!

**Comment by youknowone** [comment]
Is this PR actually intended to be merged?

It seems comments are not about the actual codes but about patch's changes. Shouldn't it be deleted for learners?

**Pull request merged**